src/App.tsx

```diff
@@ -1,61 +1,164 @@
 import './App.scss';
+import { ChangeEvent, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+let newTaskId = 0;
+
+const getUser = (userId: number): User | null => {
+  const foundUser = usersFromServer.find((user) => user.id === userId);
+
+  return foundUser || null;
+};
+
+const todos: Todo[] = todosFromServer.map((todo) => {
+  if (todo.id >= newTaskId) {
+    newTaskId = todo.id + 1;
+  }
+
+  return {
+    ...todo,
+    user: getUser(todo.id),
+  };
+});
 
 export const App = () => {
+  const [todosList, addTodo] = useState(todos);
```

Comment:
you don't have addTodo logic by default
```suggestion
  const [todosList, changeTodosList] = useState(todos);
```